packages/algob/src/lib/algo-operator.ts

```diff
@@ -1,6 +1,6 @@
 import { types as rtypes } from "@algo-builder/runtime";
 import { BuilderError, ERRORS, tx as webTx, types as wtypes } from "@algo-builder/web";
-import algosdk, { LogicSigAccount, modelsv2 } from "algosdk";
+import algosdk, { getApplicationAddress, LogicSigAccount, modelsv2 } from "algosdk";
```

Comment:
Can we rename to `getAppAddress`?

Reply:
Where is this defined? Probably some file is missing in this PR?

Reply:
it is defined in the js-sdk: [https://github.com/algorand/js-algorand-sdk/blob/develop/src/encoding/address.ts#L187](https://github.com/algorand/js-algorand-sdk/blob/develop/src/encoding/address.ts#L187)

Reply:
ah, sorry, didn't look at the end of the statement ;)